lib/resources/r4/related_person_patch.yaml

```diff
@@ -65,6 +65,40 @@ operations:
         <li>If `line` is provided, only the first four values are used. Additional values will be ignored.</li>
       </ul>
 
+  - name: add-identifier
+    path: /identifier/-
+    operation: add
+    type: Identifier
+    url: https://hl7.org/fhir/R4/patient-definitions.html#RelatedPerson.identifier
+    description: Add a new identifier for the related person.
+    example: |
+      {
+        "path": "/identifier/-",
+        "op": "add",
+        "value": {
+          "type": {
+            "coding": [
+              {
+                "code": "MR",
+                "system": "http://hl7.org/fhir/v2/0203"
+              }
+            ]
+          },
+          "system": "urn:oid:1.1.1.1.1.1",
+          "value": "&lt;unique identifier&gt;",
+          "period": {
+            "start": "2016-01-02T00:00:00-05:00",
+            "end": "2020-01-02T00:00:00-05:00"
+          }
+        }
+      }
+    note: >
+      <ul>
+        <li>Must not contain the `use` fields.</li>
+        <li>Must contain the `type`, `system` and `value` fields.</li>
```

Comment:
Should we add a comma such that it reads "...the `type`, `system`, and `value`..."? Just a grammatical nit. I prefer the oxford comma, but lmk if we should leave as-is.

Reply:
hmmm I think I prefer it the way it is but since there are two votes for this I changed it [here](https://github.com/cerner/fhir.cerner.com/pull/555/commits/ada4136825e1bad402aebcac6562da87c1340e33#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dL98)